src/interfaces/mining.h

```diff
@@ -42,9 +43,13 @@ class Mining
      *
      * @param[in] script_pub_key the coinbase output
      * @param[in] use_mempool set false to omit mempool transactions
+     * @param[in] coinbase_max_addition_weight maximum additional weight which the pool will add to the coinbase transaction
```

Comment:
I take it you wanted to rename it to `coinbase_max_additional_weight` instead

Reply:
Thanks.